src/tests-event-replay/import-export-tests.ts

```diff
@@ -95,7 +99,11 @@ describe('import/export tests', () => {
     await expect(databaseHasData({ ignoreMigrationTables: true })).resolves.toBe(false);
   });
 
-  afterEach(async () => {
-    await db?.close();
+  test('Bns import occurs', async () => {
+    process.env.BNS_IMPORT_DIR = 'src/tests-bns/import-test-files';
+    await importEventsFromTsv('src/tests-event-replay/tsv/mocknet.tsv', 'archival', true, true);
+    const configState = await db.getConfigState();
+    expect(configState.bns_names_onchain_imported).toBe(true);
+    expect(configState.bns_subdomains_imported).toBe(true);
```

Comment:
Do any of these tests validate that the imported names are correct?

Reply:
No, I can look into that, but I figured I didn't make any changes to the import process and that there would be other tests in place for this

Reply:
We have the v1 import test and additionally the config wouldnt update if any part of the import process failed so we should be good!